spec/api/search_spec.rb

```diff
@@ -175,7 +175,7 @@ def check_sort(sort_key, sort_order, expected_thread_indexes)
         end
 
         it "by activity" do
-          asc_order = [0, 2, 5, 1, 3, 4]
+          asc_order = [0, 1, 2, 3, 4, 5]
```

Comment:
Again, do we have a test somewhere for updating activity/sort order after create comment/response?